documentation/src/main/asciidoc/cassandraclient.adoc

```diff
@@ -0,0 +1,299 @@
+= Quarkus - Using the Cassandra Client
+
+Cassandra is a well known NoSQL Database that is widely used.
```

Comment:
This is a bit vague. Even if this is just a brief introductory description, we could be more specific. The first sentence of the Wikipedia article sounds like a good starting point:

> Apache Cassandra is a free and open-source, distributed, wide column store, NoSQL database management system designed to handle large amounts of data across many commodity servers, providing high availability with no single point of failure.